rules/network/initial_access_unsecure_elasticsearch_node.toml

```diff
@@ -38,9 +38,10 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-(event.dataset: network_traffic.http OR (event.category: network_traffic AND network.protocol: http)) AND
-    status:OK AND destination.port:9200 AND network.direction:inbound AND NOT http.response.headers.content-type:"image/x-icon" AND NOT
-    _exists_:http.request.headers.authorization
+(event.dataset:network_traffic.http or (event.category:network_traffic and network.protocol:http)) and
```

Comment:
Because our query validation for integrations occurs _separately_ from the base query validation (ECS + beats diff), there is a conflict where a rule cannot satisfy both validations. This is the first rule with an integration-specific field (that I am aware of), which results in it
* failing base query validation because the fields do not exist in ECS or beats
* passing integration validation

As a test, even if we added the fields to non-ecs-schema (as a bad hack), the rule still fails because the ECS/beats field is not known to the integration

Integration only fields:
* `http.response.status_phrase`
* `network_traffic.http.request.headers.authorization`

Unknown to integration
* `http.request.headers.authorization`
* `status`

With the existing process, the only way to rectify this is to split the rule, however that is not sustainable. We need to review this for potential refactor.

--- 

attempted cross-compatible query, which fails based on the above

```sql
(event.dataset:network_traffic.http or (event.category:network_traffic and network.protocol:http)) and
    (status:ok or http.response.status_phrase:ok) and
    destination.port:9200 and network.direction:inbound and
    not http.response.mime_type:"image/x-icon" and
    not (network_traffic.http.request.headers.authorization:* or http.request.headers.authorization:*)
```

Reply:
Reference the description above for further details and coexisting bugs

Reply:
Thanks Justin for bringing this to our attention! I will have to dive a bit deeper, but I recall that we specifically allow it to validate ECS+beats OR integrations. 

https://github.com/elastic/detection-rules/blob/a6c5cfc4187f2ae43e4e8ae526ec701be4dcfae1/detection_rules/rule_validators.py#L48-L70

If you review this code, `validation_checks` holds the state of ECS+Beats OR Integration validation checks. If both fail, then yes the rule should be reviewed because neither the integration's schema nor ECS, Non-ECS or Beats have the fields or the query is incorrect.